2-multitenant/envs/development/variables.tf

```
@@ -31,6 +31,7 @@ variable "envs" {
 variable "apps" {
   description = "Applications"
   type = map(object({
+    acronyms         = string
```

Should this be singular (as it's just a string)?

```suggestion
    acronym          = string
```